### PR TITLE
Add changelog and remove old buildguide

### PR DESCRIPTION
**Pull request by kn0wmad**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	- Developer defined
+	- Internal, dependencies, and/or external
+- Full Embassy backup (diff-based)
+- External drive support/requirement
+	- Single at first
+	- Groundwork for extension and mirror drives
+- Disk encryption
+	- Random key encrypted with static value
+	- Groundwork for swapping static value with chosen password
+- Session-based auth
```
Comment (dr-bonez) [inline changelog.md:45]: this is not new

Comment (dr-bonez) [reply changelog.md:45]: to be clear: only session-based auth is not new. The way the comment shows is misleading.

Comment (MattDHill) [reply changelog.md:45]: As discussed, let's mark as "Session Management" as opposed to session-based auth

Comment (kn0wmad) [reply changelog.md:45]: Thanks for clarifying, will edit

**Comment by ProofOfKeags** [comment]
was this merged in a different PR?

**Comment by kn0wmad** [comment]
I borked this and changelog went directly into integration/0.3 by mistake, cleared this with Aiden, should have commented here, apologies.

**Pull request closed without merging**